messaging/src/main/java/org/axonframework/eventhandling/pooled/Coordinator.java

```diff
@@ -594,6 +646,10 @@ public void run() {
                 return;
             }
 
+            if (!tokenStoreInitialized.get()) {
```

Comment:
This is now always `false` right? This if can be removed and the `initializeTokenStore` method can remove the `tokenStoreInitialized` as well if I am reading this all correctly

Reply:
No, since the taks sometimes is started again. Hence we might want to move to the coordinator start method. In which case we don't need the `tokenStoreInitialized` anymore.

Reply:
I read this wrong, sorry.